Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -20,6 +24,7 @@ public abstract class BaseWorldManager : MonoBehaviour
 
         protected abstract Vector3 SpawnLocation { get; }
         protected abstract Quaternion SpawnLookDirection { get; }
+        protected abstract void SetupAnalytics();
```

Comment:
I'm thinking that it might be cleaner to make this a virtual method in which it implements the `DeathAreaBehavior` Initialization and just overrides the method to do nothing for the test scenes (`sandbox`, `GunRange `and etc). Since it looks like this is the primary logic that all proper scenes will most likely have.

```suggestion
        protected virtual void SetupAnalytics()
        {
            FindObjectsOfType<DeathAreaBehaviour>().ForEach(area => area.Initialize(_worldNameAnalytics));
        }
```

Reply:
I basically removed the setup analytics completely. just better to let each component handle their own analytics if it's needed